ui/src/journals/containers/DetailPageContainer.tsx

```diff
@@ -102,7 +100,7 @@ export const DetailPage = ({ result, isCatalogerLoggedIn }: { result: Journal, i
             <Row className="mt2">
               <Col>
                 <Link
-                  to={`${LITERATURE}?sort=mostrecent&size=25&page=1&q=publication_info.journal_title.raw:"${shortTitle}"`}
+                  to={`${LITERATURE}?sort=mostrecent&size=25&page=1&q=publication_info.journal_title:"${shortTitle}"`}
```

Comment:
why we change it?

Reply:
@MJedr noticed [above](https://github.com/inspirehep/inspirehep/pull/2697#discussion_r999190392) that correct query should be without raw (it works with and without, but since I changed it in JournalPapers.tsx I changed here as well to be consistent)